pkg/kubelet/qos/util/qos.go

```diff
@@ -19,12 +19,7 @@ package util
 import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
-)
-
-const (
-	Guaranteed = "Guaranteed"
-	Burstable  = "Burstable"
-	BestEffort = "BestEffort"
+	qostypes "k8s.io/kubernetes/pkg/kubelet/qos/types"
```

Comment:
Yeah.  I see no benefit in util at all so if we are refactoring lets just
drop it and combine to one package.

On Wednesday, June 22, 2016, Buddha Prakash <notifications@github.com>
wrote:

> In pkg/kubelet/qos/util/qos.go
> <https://github.com/kubernetes/kubernetes/pull/27749#discussion_r68155316>
> :
>
> > @@ -19,12 +19,7 @@ package util
> >  import (
> >  	"k8s.io/kubernetes/pkg/api"
> >  	"k8s.io/kubernetes/pkg/api/resource"
> > -)
> > -
> > -const (
> > -	Guaranteed = "Guaranteed"
> > -	Burstable  = "Burstable"
> > -	BestEffort = "BestEffort"
> > +	qostypes "k8s.io/kubernetes/pkg/kubelet/qos/types"
>
> Yeah we will have import cycles because of the way the code is currently
> structured, but making the changes you suggest would make sense.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27749/files/9b264a0fe89b04d775a58996df07e135b030cec6#r68155316>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8dbFcuuGxAWNjPwG2gvyDFQbkDNiltks5qOcjTgaJpZM4I6SZg>
> .
>